### PR TITLE
Moved failing tests to the flaky suite on Jenkins

### PR DESCRIPTION
**Pull request by piosz**

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -101,6 +101,9 @@ GCE_FLAKY_TESTS=(
     "DaemonRestart"
     "ResourceUsage"
     "monotonically\sincreasing\srestart\scount"
+    "then\sschedule\sit\son\sanother\shost" # pd.go
+    "should\sbe\sable\sto\schange\sthe\stype\sand\snodeport\ssettings\sof\sa\sservice" # service.go
+    "allows\sscheduling\sof\spods\son\sa\sminion\safter\sit\srejoins\sthe\scluster" # resize_nodes.go
```
Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: Can you add issue number to each of the bugs? These are respectively:
#13257 #13032 #13258

**Comment by wojtek-t** [comment]
cc @quinton-hoole

**Comment by wojtek-t** [comment]
cc @kubernetes/goog-testing

File: hack/jenkins/e2e.sh (hunk 1 above)
Comment (piosz) [inline hack/jenkins/e2e.sh]: Done

**Comment by wojtek-t** [comment]
LGTM

**Comment by piosz** [comment]
@k8s-bot test this please

**Pull request merged**